tests/api_tests/test_participant_summary_api.py

```diff
@@ -506,6 +506,11 @@ def test_invalid_filters_return(self):
         response_no_filter = self.send_get("ParticipantSummary")
         self.assertEqual(len(response_no_filter['entry']), num_summary)
 
+    def test_access_with_curation_role(self):
+        self.overwrite_test_user_roles([CURATION])
+        response = self.send_get("ParticipantSummary")
+        self.assertBundle([], response)
+
```

Comment:
Can you generate some `participant_summaries` and make sure this role will return them instead of returning `[]`

Reply:
Done :)

Reply:
🔥